testdata/test.yaml

```diff
@@ -0,0 +1,11 @@
+schemaVersion: 2.0.0
+
+fileExistenceTests:
+  - name: foo
+    path: "/foo"
+    shouldExist: true
+
+fileContentTests:
```

Comment:
I think I'd like to see us add `exec` tests for the `lang_image` rules.

Reply:
yeah I think that's a good use case. I'm going to add tests in a follow up PR, this one is getting a bit cluttered and I'll need to fiddle with the bazel CI a bit which I don't want to do here.